website/server/models/user/methods.js

```diff
@@ -20,6 +20,49 @@ schema.methods.getGroups = function getUserGroups () {
   return userGroups;
 };
 
+// Get an array of error message keys that would be thrown if the given interaction was attempted
```

Comment:
I've thought about that, but I'm not sure it's the way to go. I'd rather change the error reporting code to be able to report multiple issues -- the new API function already does that.

Reporting only one when there are multiple seems to me to give a skewed view of the situation, kinda like "fix this one error and it will work", while reporting all gives the full view -- but maybe I'm reading too much into things here.